menas/ui/service/GenericService.js

```diff
@@ -66,10 +66,18 @@ var GenericService = new function () {
     })
   };
 
+  this.isEmpty = function (str) {
+    return !str || str === "";
```

Comment:
The second part is unneeded I think

![image](https://user-images.githubusercontent.com/17467768/60012225-b565c400-967b-11e9-8ab4-b2e2d257fa82.png)

Reply:
Holy crap, JS takes a lot of liberties in interpreting things as a boolean lol